public/js/pimcore/object/tags/time.js

```diff
@@ -29,18 +29,28 @@ pimcore.object.tags.time = Class.create(pimcore.object.tags.abstract, {
     },
 
     getLayoutEdit: function () {
-        this.component = new Ext.form.TimeField({
+        const options = new Ext.form.TimeField({
```

Comment:
I don't think we want to create a TimeField at this line, as we create one on line 53 already.